### ISSUE TITLE
Docker-beta update modal dialogs, enough is enough ! 

### ISSUE THREAD

**Issue by tjstebbing** [moby/moby#25639]
Hey docker, great project, I love your work but we need to talk. 

Don't get me wrong, docker is important to my day to day, I run docker-beta because I want your best stuff. But the in-your-face dialogs on OSX are a bit much when there's an update every second day. 

First there's one to ask if I want to update, that's pretty reasonable as I have 'check for updates' enabled, this one's totally fine!

Next comes the progress modal that on OSX floats over all else for the entire duration of the download. Can we please consider allowing this one to be backgrounded?, or dismissed? or replaced with an elegant downloading indicator on the docker-dock?
<img width="36" alt="screen shot 2016-08-12 at 8 29 42 am" src="https://cloud.githubusercontent.com/assets/551374/17607117/f5b8bc54-6066-11e6-94db-ae3b216e83c5.png">

Then once it's done I get asked to install and relaunch, again, not terrible but when you're busy working, all of these are adding up to being _annoying_. Can't we assume that if I've said yes to the upgrade I actually want to upgrade once it's downloaded? if I say no here it _throws out my downloaded update_ anyway so I'm pretty sure I'm saying yes.

So at this point I'm really hoping I'm home-free, I've clicked, I've accepted, I'm trying to get some work done and BAM

<img width="383" alt="screen shot 2016-08-12 at 8 13 14 am" src="https://cloud.githubusercontent.com/assets/551374/17607174/776a61da-6067-11e6-98cc-c6d96b21bbdc.png">

ZOMGOSH WTH IS THIS D:  It covers half my screen D:

And just to make extra sure I know that I've updated:

<img width="413" alt="screen shot 2016-08-12 at 8 13 20 am" src="https://cloud.githubusercontent.com/assets/551374/17607190/9e90df96-6067-11e6-93e3-c763078423f9.png">

Boop! this little critter :D

In conclusion, considering that docker-beta updates are so frequent, couldn't this all be reduced to some elegant stateful dock indicator? 

Thanks!

**Comment by justincormack** [comment]
Thanks for complaining!

As I work on Docker for Mac I am on the internal dev channel, which means I get notifications every time our CI system finishes a build. So I get annoyed a lot... I think that means I don't realise how much our users get annoyed. 

I definitely agree there is a lot of room for improvement.

**Comment by thaJeztah** [comment]
There's an issue (3274) on the internal issue tracker "whishlist"; I agree it can be, erm, annoying if you get multiple updates; here's the discussion we had there

<img width="792" alt="screen shot 2016-08-12 at 01 56 04" src="https://cloud.githubusercontent.com/assets/1804568/17608917/3e6ec722-6030-11e6-8018-b1dd3c252756.png">

**Comment by pomke** [comment]
Thanks for getting back to me so quickly :D   I'm not one to complain for the sake of complaining, I hope this is taken in the spirit it was intended :) I do think things like this are important, I know it's easy to prioritise usability improvements low in the backlog (we all do it every day) but small things make a big difference!  

Keep on Rockin'

**Comment by thaJeztah** [comment]
Let me close this ticket for now, as it looks like it went stale.
